core/src/main/java/org/opensearch/sql/planner/physical/RemoveOperator.java

```diff
@@ -24,26 +23,19 @@
 import org.opensearch.sql.data.model.ExprValueUtils;
 import org.opensearch.sql.expression.ReferenceExpression;
 
-/**
- * Remove the fields specified in {@link RemoveOperator#removeList} from input.
- */
+/** Remove the fields specified in {@link RemoveOperator#removeList} from input. */
 @ToString
 @EqualsAndHashCode(callSuper = false)
 public class RemoveOperator extends PhysicalPlan {
-  @Getter
-  private final PhysicalPlan input;
-  @Getter
-  private final Set<ReferenceExpression> removeList;
-  @ToString.Exclude
-  @EqualsAndHashCode.Exclude
-  private final Set<String> nameRemoveList;
+  @Getter private final PhysicalPlan input;
+  @Getter private final Set<ReferenceExpression> removeList;
+  @ToString.Exclude @EqualsAndHashCode.Exclude private final Set<String> nameRemoveList;
```

Comment:
How come the multiple annotations are put on one line here while multiple annotations for the function above are left on multiple lines.

Reply:
You'll have to look at the rules, but I think the idea is that classes aren't indented so the annotations are put above, while functions and class variables are already indented, so it doesn't hurt to indent them more...?